packages/gatsby/src/redux/reducers/webpack-compilation-hash.ts

```diff
@@ -0,0 +1,13 @@
+import { IGatsbyState, ISetWebpackCompilationHashAction } from "../types"
+
+export const webpackCompilationHashReducer = (
+  state: IGatsbyState["webpackCompilationHash"] = ``,
+  action: ISetWebpackCompilationHashAction
```

Comment:
this should be `ActionsUnion` as every reducer gets called for dispatches of any type.